package.json

```diff
@@ -8,7 +8,7 @@
     "test": "mocha"
   },
   "author": "Kevin Zhuang",
-  "license": "ISC",
+  "license": "BSD",
```

Comment:
According to[ package.json document](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#license) and [the full list of SPDX license IDs](https://spdx.org/licenses/), it should be `BSD-3-Clause`